cmd/mayactl/app/command/stats_helper_test.go

```diff
@@ -63,16 +63,6 @@ func TestGetVolDetails(t *testing.T) {
 			err:  nil,
 			addr: "MAPI_ADDR",
 		},
-		"MAPIADDRNotSet": {
-			volumeName: "vol",
```

Comment:
What will be default scenario, how are you handling that ?
is there any test case for such scenario ?

Reply:
Its getting handled in command.go file I think handling it everywhere is just repetition